resources/prosody-plugins/mod_muc_password_check.lua

```diff
@@ -0,0 +1,218 @@
+local jid = require "util.jid";
+local json = require "util.json";
+local async_handler_wrapper = module:require "util".async_handler_wrapper;
+local room_jid_match_rewrite = module:require "util".room_jid_match_rewrite;
+local starts_with = module:require "util".starts_with;
+local token_util = module:require "token/util".new(module);
+local inspect = require "inspect";
+local parse = require "net.url".parseQuery;
+
+
+-- option to enable/disable room API token verifications
+local enableTokenVerification
+= module:get_option_boolean("enable_password_token_verification", true);
```

Comment:
When would someone want this to be false? If the API is not exposed?

Reply:
Someone may want to use it without the jwt authentication part ... in other virtualhosts and handle exposing it another way ...

Reply:
I will drop it, to simplify the code :)